account_invoice_validate_send_email/models/account_invoice.py

```diff
@@ -18,6 +18,12 @@ class AccountInvoice(models.Model):
     )
     web_url = fields.Char()
 
+    yamato_slip_number = fields.Text(
+        "Yamato Slip Number",
+        help="Delivery Slip Number of Yamato Transport CO.,.",
+        compute="_compute_yamato_slip_number",
+    )
```

Comment:
The naming is not good. It's not a good idea have a field specifically for Yamato when we take the compute the value from a generic/standard field.

Reply:
I renamed field Slip Number.